server/src/main/scala/org/http4s/server/middleware/CSRF.scala

```diff
@@ -340,7 +343,8 @@ object CSRF {
     def withOnFailure(onFailure: Response[G]): CSRFBuilder[F, G] = copy(onFailure = onFailure)
     def withCreateIfNotFound(createIfNotFound: Boolean): CSRFBuilder[F, G] =
       copy(createIfNotFound = createIfNotFound)
-    def withKey(key: SecretKey): CSRFBuilder[F, G] = copy(key = key)
+    def withKey(key: crypto.SecretKey): CSRFBuilder[F, G] =
```

Comment:
I'm surprised and suspicious that this passed MiMa.

It's also introducing a `private[http4s]` into the public API.  How is anybody else going to be able to call it?

Reply:
Ooph, this is just rough on the eyes I think. But it passes MiMa because it actually refers to `import javax.crypto`.

Reply:
Let me write those out as FQCN so we can sleep easy.

Reply:
There. `crypto` made more sense when the package was `bobcats` and not `crypto`.